proto/wtools/abase/l0/l1.test/ArgumentsArray.test.s

```diff
@@ -327,6 +368,126 @@ function from( test )
   test.shouldThrowErrorSync( () => _.argumentsArray.from( {} ) );
   test.shouldThrowErrorSync( () => _.argumentsArray.from( 'wrong' ) );
 }
+//
+
+function isPerformance( test )
+{
+  /* Average of 10 runs of 5 million ierations of 8 input variants
+  ╔═══════════════════╤═════╤═════╤══════════════╤═════╤═══════╤══════════════════╤════════════╗
+  ║                   │  is │isOld│isUsingFunctor│ like│likeOld│likeUsingIsFunctor│likeUnfolded║
+  ╟───────────────────┼─────┼─────┼──────────────┼─────┼───────┼──────────────────┼────────────╢
+  ║ **Njs : v10.24.1**│0.594│0.594│     0.598    │0.767│ 0.767 │       0.780      │    0.612   ║
```

Comment:
![image](https://user-images.githubusercontent.com/7414255/120446625-64e06080-c392-11eb-870d-4f63714b85b4.png)
Why same result?

Reply:
is and like routines are still on original version, isOld and likeOld is same as is. Should i remove isOld and likeOld Columns or should i duplicate is and like functions to make isOld and likeOld functions?